cluster/images/kubelet/Makefile

```diff
@@ -1,21 +1,21 @@
-# build the kubelet image.
+# Build the kubelet image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-# TODO: figure out the best base image
-# TODO: figure out how to best source the nsenter binary, if necessary
-# TODO: figure out how to make a single source of version info for these
-#       images
-VERSION=v0.16
-BIN_DIR=bin/linux/amd64
-LOCAL_OUTPUT_DIR=../../../_output/local
+REGISTRY?="gcr.io/google_containers"
+ARCH=amd64
 
-release: clean
-	curl -O https://storage.googleapis.com/kubernetes-release/release/${VERSION}/${BIN_DIR}/kubelet
-	docker build -t gcr.io/google_containers/kubelet:${VERSION} .
-	gcloud docker push gcr.io/google_containers/kubelet:${VERSION}
-
-local: clean
-	cp ${LOCAL_OUTPUT_DIR}/${BIN_DIR}/kubelet .
-	docker build -t gcr.io/google_containers/kubelet .
+build:
+ifndef REGISTRY
+    $(error REGISTRY is undefined)
+endif
+ifndef VERSION
+    $(error VERSION is undefined)
+endif
+	cp ../../../_output/dockerized/bin/linux/${ARCH}/kubelet .
```

Comment:
Never mind, I was just thinking of how it had been earlier, when the `Makefile` `curl`ed for `hyperkube`. However this is the way to go